Public/Sdk/Public/Managed/Frameworks/netcoreapp2.2/netcoreapp2.2.dsc

```diff
@@ -16,8 +16,9 @@ function macOSRuntimeExtensions(file: File): boolean {
 
 function ignoredAssembly(file: File): boolean {
     let f = file.name;
-    return f.equals(a`System.Security.AccessControl.dll`)
-    || f.equals(a`System.Security.Principal.Windows.dll`);
+    // We skip deploying those files from the .NET Core package as we need those very assemblies from their dedicated package
+    // to compile our platform abstraction layer, which depends on datatypes present only in the dedicated packages
+    return f.equals(a`System.Security.AccessControl.dll`) || f.equals(a`System.Security.Principal.Windows.dll`);

```

Comment:
Do you know why use `.equals()` here and not `===`?  I'd expect `===` to work just the same.

Can you also rename `f` to `fileName`?

Reply:
I guess `.equals()` just because of jumping between DScript and C# - renamed it.